src/components/TokenBalance.js

```diff
@@ -0,0 +1,33 @@
+import { useWallet } from "@solana/wallet-adapter-react";
+import { useEffect, useState } from "react";
+import { ANTI_TOKEN_MINT, getTokenBalance, PRO_TOKEN_MINT } from "../utils/solana";
+
+const TokenBalance = () => {
+  const wallet = useWallet()
+  const [antiBalance, setAntiBalance] = useState(0)
+  const [proBalance, setProBalance] = useState(0)
+
+  useEffect(() => {
+    const checkBalance = async () => {
+      const [antiBalanceResult, proBalanceResult] = await Promise.all(
+        [
+          getTokenBalance(wallet.publicKey, ANTI_TOKEN_MINT),
+          getTokenBalance(wallet.publicKey, PRO_TOKEN_MINT)
+        ]
+      )
+      setAntiBalance(antiBalanceResult)
+      setProBalance(proBalanceResult)
+    }
+    if (wallet.publicKey)
```

Comment:
either braces here or having the code in a single line would be preferred to prevent any bugs in the future

```suggestion
    if (wallet.publicKey) checkBalance()
```